packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -284,6 +284,7 @@ export const installGovAndPSMContracts = async ({
       binaryVoteCounter,
       auction,
       psm,
+      VaultFactory: vaultFactory,
```

Comment:
why is startPSM starting the vaultFactory? if proceeding with these, please rename the module